pr_testing/run-pr-unittests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -ex
+source $(dirname $0)/setup-pr-test-env.sh
+
+echo 'UNIT_TEST_RESULTS;NOTRUN' > ${RESULTS_DIR}/unittestGPU.txt
+if [ "X$DO_TESTS" = Xtrue ]; then
+  rm -f ${RESULTS_DIR}/unittestGPU.txt
+  mark_commit_status_all_prs 'unittests/gpu' 'pending' -u "${BUILD_URL}" -d "Running tests" || true
+  echo '--------------------------------------'
+  mkdir -p $WORKSPACE/gpuUnitTests
+  let UT_TIMEOUT=7200+${CMSSW_PKG_COUNT}*20
+  UTESTS_CMD="USER_UNIT_TESTS=cuda timeout ${UT_TIMEOUT} scram b -v -k -j ${NCPU}  runtests "
+  echo "LD_LIBRARY_PATH: ${LD_LIBRARY_PATH}"
+  scram build echo_LD_LIBRARY_PATH || true
+  cms_major=$(echo ${CMSSW_IB} | cut -d_ -f2)
+  cms_minor=$(echo ${CMSSW_IB} | cut -d_ -f3)
+  cms_ver="$(echo 00${cms_major} | sed -E 's|^.*(..)$|\1|')$(echo 00${cms_minor} | sed -E 's|^.*(..)$|\1|')"
+  if [ $cms_ver -ge 1301 ] ; then
+    find $CMSSW_BASE/src -type d | grep -v '/__pycache__/*' | xargs chmod -w
+  fi
+  echo $UTESTS_CMD > $WORKSPACE/gpuUnitTests/log.txt
+  (eval $UTESTS_CMD && echo 'ALL_OK') > $WORKSPACE/gpuUnitTests/log.txt 2>&1 || true
+  echo 'END OF UNIT TESTS'
+  echo '--------------------------------------'
+  if [ $cms_ver -ge 1301 ] ; then
+    find $CMSSW_BASE/src -type d | xargs chmod +w
+  fi
+
+  TEST_ERRORS=$(grep -ai 'had errors\|recipe for target' $WORKSPACE/gpuUnitTests/log.txt | sed "s|'||g;s|.*recipe for target *||;s|.*unittests_|---> test |;s| failed$| timeout|" || true)
+  TEST_ERRORS=`grep -ai "had errors" $WORKSPACE/gpuUnitTests/log.txt` || true
+  GENERAL_ERRORS=`grep -a "ALL_OK" $WORKSPACE/gpuUnitTests/log.txt` || true
+
+  if [ "X$TEST_ERRORS" != "X" -o "X$GENERAL_ERRORS" = "X" ]; then
+    echo "Errors in the gpu unit tests"
+    echo 'UNIT_TEST_RESULTS;ERROR,Unit Tests,See Log,gpuUnitTests' >> ${RESULTS_DIR}/unittestGPU.txt
```

Comment:
`UNIT_TEST_RESULTS` is unit key, for gpu unit tests it should be `GPU_UNIT_TEST_RESULTS` otherwise it will override the results of cpu unit tests